marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/addCustomStepToFlow.cy.tsx

```diff
@@ -90,4 +90,17 @@ describe("Add Custom step to a flow", () => {
     runPage.verifyStepRunResult(stepName, "success");
     runPage.closeFlowStatusModal(flowName);
   });
+});
+
+describe("Check spinner", () => {
+  it("Check Spinner when page is loading", () => {
+    cy.log(Cypress.config("baseUrl"));
+    cy.visit("/");
+    cy.contains(Application.title);
+    cy.loginAsDeveloper().withRequest();
+    cy.visit("/tiles/curate");
+    cy.findByTestId("spinner");
+    cy.wait(4000);
```

Comment:
Can we try using the function we have already to avoid the 4000ms wait?

Pulled from Browse.tsx:13:

  waitForSpinnerToDisappear() {
    cy.waitUntil(() => this.getSpinner().should("have.length", 0, {timeout: 30000}));
    cy.waitForAsyncRequest();
  }

Reply:
Updated 🛠